disk-deactivate

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+# dependencies: jq util-linux lvm2 mdadm zfs
+disk=$1
+
+lsblk --output-all --json | jq -r --arg disk_to_clear "$disk" '
```

Comment:
I think at this lenght, the jq script could become its own file. Not sure if editors have syntax highlighting for that....

Reply:
well it mostly is it's own file? or should we move the jq code to another extra file?

Reply:
Yep. I mean the jq script.